src/components/confirmation-box/confirmation-box.tsx

```diff
@@ -62,6 +72,13 @@ const Box = styled.div<Pick<ConfirmationBoxProps, "open" | "preventAnimation">>`
     border-radius: 0;
   }
 
+  p {
+    margin: 0 0 ${({ theme }) => theme.spaces.xs};
+    line-height: ${({ theme }) => theme.lineHeights.copy};
+    text-align: center;
+    word-break: break-word;
+  }
+
```

Comment:
This allows to style the text inside the children without having to declare new Styled Components each time (as it is done on some compos)

Reply:
I was discussing this with Pauline (because she copied it from here). I understand that you went for this but just as a reminder, we want to avoid heritage at all cost.

It's ok for now as it is the implementation that "forces" this kind of thing, and it will be redone :).

Reply:
What do you mean by "heritage" ?
From my perspective (and we talked about it) this component is ... well... let's say clunky 😁 
So we just have to use a `<p></p>` for the confirmation question and it is kind of "normal" i'd say to define its style inside the component that will display it, right?

But as I discussed it with Pauline too, I'd really like to make this component differently because in the end it should be something like this:
```ts
<ConfirmationBox
  confirmButtonText="confirm..."
  cancelButtonText="cancel"
  confirmationText="..."
  action={functionThatShouldBeUsedOnConfirmation}
/>
```

Reply:
Because for now we are just "forced" to declare the whole content each time and it is "risky" 😅

Reply:
Yeah I agree with this :)